_test/page.getter.test.php

```diff
@@ -48,7 +48,7 @@ public function test_get_pages_without_title_gets_pages_sorted_by_time()
         $pagesHaveNoTitle = ['page0', 'page1', 'page2'];
         foreach ($pagesHaveNoTitle as $pageHasNoTitle) {
             Helper::savePages([$pageHasNoTitle], false);
-            sleep(60);
+            sleep(1);
```

Comment:
That's a 60x performance increase. Nice 😃

Reply:
It is ;) :racehorse: :white_check_mark: